docs/workflows/create_sync_publish.rst

```diff
@@ -41,6 +41,20 @@ By default ``policy='immediate`` which means that all the content is downloaded
 Specify ``policy='on_demand'`` to make synchronization of a repository faster and only
 to download RPMs whenever they are requested by clients.
 
+Also, you can specify ``client_cert`` and ``client_key`` if your remote require authorization with a certificate.
+
+.. code:: shell
+
+    http --form POST $BASE_ADDR/pulp/api/v3/remotes/rpm/rpm/ \
```

Comment:
In my testing with certguard the `--form`` didn't work. Did this actually work for you?

Reply:
It did. I extended command in scripts and it works well for me. 
`http --form POST $BASE_ADDR/pulp/api/v3/remotes/rpm/rpm/ 
    name='bar' 
    url="$URL" 
    policy='on_demand' 
    client_cert=@./cert.crt 
    client_key=@./cert.key 
    ca_cert=@./ca_cert.pem
`